### ISSUE TITLE
HTML-escaped error messages

### ISSUE THREAD

**Issue by lovasoa** [intellij-rust/intellij-rust#5241]
Hello,
  I'm not sure this is the right place to report this, but I am having issues with error messages that contain special characters: they are escaped as html entities.

I see: *Error:(340, 21) mismatched types [E0308]expected `Simd&#60;[u8; 8]&#62;`, found `Simd&#60;[i32; 8]&#62;`*

Instead of *Error:(340, 21) mismatched types [E0308]expected `Simd<[u8; 8]>;`, found `Simd<[i32; 8]>`*

## Environment

* **IntelliJ Rust plugin version:** 0.2.119.2201-201
* **Rust toolchain version:** 1.42.0 (b8cedc004 2020-03-09) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2020.1 Community Edition (IC-201.6668.121)
* **Operating system:** macOS 10.15.3

## Problem description

<img width="940" alt="image" src="https://user-images.githubusercontent.com/552629/79214256-ce2da900-7e4a-11ea-83b8-8971af9b01e8.png">

## Steps to reproduce

```rust
fn stbi_clamp_simd(x: i32x8) -> u8x8
{
    u8x8::from_cast(x.max(i32x8::splat(0)).min(i32x8::splat(255)))
}
```

**Comment by hi2u** [comment]
I'm still seeing HTML entities in the "Problems" panel:

![2020-11-19_024957-Window](https://user-images.githubusercontent.com/9689599/99560923-25fc4600-2a12-11eb-9673-acb1e33dd6b7.png)


* Rust plugin version: 0.3.135.3484-203

* IntelliJ IDEA 2020.3 Beta (Ultimate Edition)
```
- Build #IU-203.5784.10, built on November 10, 2020
- IntelliJ IDEA EAP User
- Runtime version: 11.0.9+11-b1145.10 amd64
- VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
- Windows 10 10.0
- GC: ParNew, ConcurrentMarkSweep
- Memory: 1981M
- Cores: 8
- Non-Bundled Plugins: Statistic, com.4lex4.intellij.solarized, intellij.haskell, com.markskelton.one-dark-theme, in.xiv.cs, org.toml.lang, com.jetbrains.php, training, org.rust.lang, com.samdark.intellij-visual-studio-code-dark-plus
```

**Comment by Undin** [comment]
@hi2u it will be fixed by #6536
